main/zoe/guide/contracts/atomic-swap.md

```diff
@@ -2,130 +2,80 @@
 
 <Zoe-Version/>
 
-##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/958a2c0a3dec38bdba2234934119ea2c28958262/packages/zoe/src/contracts/atomicSwap.js) (Last updated: 4/22/2020)
-##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
-
-If I want to trade one kind of asset for another kind, I could send
-you the asset and ask you to send me the other kind back. But, you
-could choose to behave opportunistically: receive my asset and give
-nothing back. To solve this problem, swap contracts allow users to
-securely trade one kind of digital asset for another kind, leveraging Zoe for
-escrow and offer safety. At no time does any user have the ability to
-behave opportunistically.
-
-In the `atomicSwap` contract, anyone can securely swap with a counterparty by escrowing digital assets with Zoe and sending an invite to the swap to the counterparty.
-
-Let's say that Alice wants to swap with Bob as the counterparty. She
-knows that the code for the swap has already been installed, so she
-can create a swap instance from the swap installation (`handle` is the
-unique, unforgeable identifier):
-
+##### [View all example contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
+
+If one party, say Alice, wants to trade one kind of asset for another kind, 
+she could send another party, Bob, the asset and ask him to send her the other
+kind back. But Alice could decide to not play fair and get Bob's asset without
+sending anything back. 
+
+To solve this problem, swap contracts let users securely trade one kind of 
+digital asset for another kind. In the `atomicSwap` contract, anyone can securely 
+swap with a counterparty by escrowing digital assets with Zoe and sending an invite 
+to the swap to the counterparty.
+
+Alice's initial offer is `{ give: { Asset: A }, want: { Price: B } }`.
+Its outcome is an invitation for the second party, Bob, who should offer
+`{ give: { Price: B }, want: { Asset: A } }`, with a `want`
+amount no greater than Alice's `give`, and a `give` amount at least as
```

Comment:
The "greater than" language was hard to parse. Maybe: For the trade to succeed, Bob shouldn't want any more than what Alice gives, and Bob should give enough to cover what Alice says she wants.